system/Cache/Handlers/FileHandler.php

```diff
@@ -21,11 +21,9 @@
 class FileHandler extends BaseHandler
 {
 	/**
-	 * Prefixed to all cache names.
-	 *
-	 * @var string
+	 * Maximum key length.
 	 */
-	protected $prefix;
+	public const MAX_KEY_LENGTH = 255;
```

Comment:
I've just learned from Symfony Filesystem of the `PHP_MAXPATHLEN` constant available since PHP5.3. Why did you not use it here instead?

Reply:
Ooo that's great! Will change.